test/end-to-end/pages/createImage.js

```diff
@@ -9,7 +9,7 @@ module.exports = class CreateImagePage extends MainPage {
 
     // ---- Root element selector ---- //
     // Create Blueprint dialog root selector
-    this.dialogRootElement = 'div[id="cmpsr-modal-crt-image"]';
+    this.dialogRootElement = `${this.dialogRootElement} #cmpsr-modal-crt-image`;
```

Comment:
I like replacing this:
`'div[id="cmpsr-modal-crt-image"]'`
with this:
`#cmpsr-modal-crt-image`

But I don't think we need the `${this.dialogRootElement}` in `${this.dialogRootElement} #cmpsr-modal-crt-image`. I think this is where `this.dialogRootElement` is getting defined to be used for other selectors below.

@henrywang Can you please confirm if this is true? Can we just use this here:
`this.dialogRootElement = '#cmpsr-modal-crt-image'`;

Reply:
I saw `${this.dialogRootElement}` used elsewhere in the file and figured it needed to be used here, would it be redundant to put it on the right side in this case?

Reply:
> would it be redundant to put it on the right side in this case?

I'm not sure what you mean by "right side"